sys/dev/usb/uvc/uvc_buf.c

```diff
@@ -0,0 +1,707 @@
+/*
+ *
+ * Copyright (c) 2024 Dell Inc.. All rights reserved.
```

Comment:
"All rights reserved." is not really needed nowadays. Also, we need a SPDX header, and `/*-` for automatic script collects license. Please refer to style(9) for more details.

Recommend:

```
/*-
 *SPDX-License-Identifier: BSD-2-Clause
 *
 * Copyright (c) 2024 Dell Inc.
 *
 * Redistribution and use in source and binary forms, with or without
...
```